docs/F-Refinements/portability.md

```diff
@@ -1,94 +1,99 @@
+- Multi-threading support
+- Improved Unicode support
+- Bounds checking interfaces
+- A create and open mode **("x")** for `fopen()`
+- Removal of `gets()`
+- Optional support for complex arithmetic
+- Optional support for variable-length arrays
 
-The optional support applies to features that had been mandatory under the C99 standard. 
+The optional support applies to features that had been mandatory under the C99 standard.
 
 ## Structured Programs
 
-Software engineers have advocated certain paradigms for developing software.  The *Structured Programming* paradigm achieves clarity by distinguishing high-level coding techniques from low-level assembly language coding techniques.  The principles of this paradigm apply to coding in many languages, including C.  They may be summarized as follows:
+Software engineers have advocated certain paradigms for developing software. The _Structured Programming_ paradigm achieves clarity by distinguishing high-level coding techniques from low-level assembly language coding techniques. The principles of this paradigm apply to coding in many languages, including C. They may be summarized as follows:
 
-* a structured program consists solely of three constructs: sequences, selections, and iterations
-* each structured construct exhibits a single entry point and a single exit point
-* statements that jump outside a construct are not allowed with certain exceptions (`switch case` constructs in the C language)
+- A structured program consists solely of three constructs: sequences, selections, and iterations
+- Each structured construct exhibits a single entry point and a single exit point
+- Statements that jump outside a construct are not allowed with certain exceptions (`switch case` constructs in the C language)
 
 The following style points highlight structured properties of program code:
 
-* indent and align code to reveal the structure of the logic
-* indent the code consistently within a structured construct
-* separate constructs by blank lines to improve clarity
+- Indent and align code to reveal the structure of the logic
+- Indent the code consistently within a structured construct
+- Separate constructs by blank lines to improve clarity
 
 ## Static Analysis (Optional)
 
-Static analysis of a program can reveal code that might not be portable.  Tools for static analysis flag suspicious code.  Some popular tools for C language programs include:
+Static analysis of a program can reveal code that might not be portable. Tools for static analysis flag suspicious code. Some popular tools for C language programs include:
 
-* lint - the original analyzer written by Stephen Johnson in 1979
-* Parasoft C/C++ test - plugins for Visual Studio and Eclipse IDEs
-* Splint - a modern open source version of lint
+- Lint - the original analyzer written by Stephen Johnson in 1979
+- Parasoft C/C++ test - plugins for Visual Studio and Eclipse IDEs
+- Splint - a modern open source version of Lint
```

Comment:
As mentioned on another comment, enclose "Splint" and "Lint" within backticks (\`Splint\` and \`Lint\`).